src/components/Nav/Nav.tsx

```diff
@@ -55,9 +59,15 @@ export class Nav extends React.Component<INavProps, INavState> implements INav {
     return this._selectedKey;
   }
 
-  private _renderLink(link: INavLink, linkIndex: number): React.ReactElement<{}> {
+  private _renderLink(link: INavLink, linkIndex: number, nestingLevel: number, hasGroupButton: boolean): React.ReactElement<{}> {
     let { onLinkClick } = this.props;
 
+    const isRtl: boolean = getRTL();
+    const paddingAfter: string = '0';
```

Comment:
Doing this in JavaScript means fewer bytes over the wire (since we don't have to generate a CSS class for each indentation level).